CMakeLists.txt

```diff
@@ -174,8 +168,6 @@ project(GSI)
     find_package(Baselibs REQUIRED)
   else()
     find_package( NetCDF COMPONENTS C Fortran REQUIRED)
```

Comment:
While compiling on Hera, the following warning message appeared:

CMake Warning (dev) at CMakeLists.txt:170 (find_package):
  Policy CMP0074 is not set: find_package uses <PackageName>_ROOT variables.
  Run "cmake --help-policy CMP0074" for policy details.  Use the cmake_policy
  command to set the policy and suppress this warning.

  Environment variable NetCDF_ROOT is set to:

    /scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/intel-18.0.5.274/impi-2018.0.4/netcdf/4.7.4

  For compatibility, CMake is ignoring the variable.
This warning is for project developers.  Use -Wno-dev to suppress it.

Please add:

cmake_policy(SET CMP0074 NEW)

to line 111:

  cmake_policy(SET CMP0009 NEW)
  cmake_policy(SET CMP0054 NEW)
  cmake_policy(SET CMP0074 NEW)
  find_package(OpenMP)

Reply:
Thank you, Mike, for your review.   The requested change was made and committed at 8578601.   Execution of `ush/build_all_cmake.sh` on WCOSS_D and Hera show that the `CMake Warning` message is no longer generated.